zcash_client_sqlite/src/wallet/init/migrations/add_transaction_views.rs

```diff
@@ -438,7 +440,11 @@ mod tests {
 
         let usk = UnifiedSpendingKey::from_seed(&network, &[0u8; 32][..], AccountId::ZERO).unwrap();
         let ufvk = usk.to_unified_full_viewing_key();
-        let (ua, _) = ufvk.default_address(UnifiedAddressRequest::unsafe_new(false, true, true));
+        let (ua, _) = ufvk.default_address(UnifiedAddressRequest::unsafe_new(
+            false,
+            true,
+            UA_TRANSPARENT,
```

Comment:
This and the other migrations are going to need to be handled carefully in future when we allow Sapling to be disabled, because the migrations are written assuming that Sapling is always available.